src/core_plugins/kibana/server/lib/crypto.js

```diff
@@ -0,0 +1,106 @@
+import crypto from 'crypto';
```

Comment:
> Shaunuk's PR description seems like a great starting point:

I really like this pattern, it almost feels like we could start asking that the pr description just link to the doc's in code